mariaDB4j/src/test/java/ch/vorburger/mariadb4j/tests/MariaDB4jSampleTutorialTest.java

```diff
@@ -89,19 +89,20 @@ public void testEmbeddedMariaDB4jWithSecurity() throws Exception {
         DBConfigurationBuilder config = DBConfigurationBuilder.newBuilder();
         config.setPort(0); // 0 => autom. detect free port
         config.setSecurityDisabled(false);
+        config.setDefaultRootPassword("root");
```

Comment:
I think we should be using the root password that is passed as an environment variable?